src/WorkspaceContext.ts

```diff
@@ -45,6 +50,13 @@ export class WorkspaceContext implements vscode.Disposable {
         this.statusItem.dispose();
     }
 
+    /** Get swift version and create WorkspaceContext */
+    static async create(extensionContext: vscode.ExtensionContext): Promise<WorkspaceContext> {
+        // get swift version and then create
```

Comment:
This seems a bit redundant

Reply:
Unfortunately in Typescript you can't have async constructors, so I need to create an async static function